src/main/java/hellos/Hellos.java

```diff
@@ -17,9 +17,13 @@ public void run() {
   }
 
   public String generateOutput() {
-    StringBuilder builder = new StringBuilder();  
+
+    StringBuilder builder = new StringBuilder();
+
```

Comment:
You probably don't want an extra blank line here. I'm not going to block it now in case you're still online and can merge quick while the opportunity exists.